api/src/pcapi/routes/backoffice_v3/tags.py

```diff
@@ -0,0 +1,121 @@
+from flask import flash
+from flask import redirect
+from flask import render_template
+from flask import url_for
+import sqlalchemy as sa
+
+import pcapi.core.criteria.models as criteria_models
+import pcapi.core.permissions.models as perm_models
+from pcapi.models import db
+
+from . import utils
+from .forms import empty as empty_forms
+from .forms import tags as tags_forms
+
+
+tags_blueprint = utils.child_backoffice_blueprint(
+    "tags",
+    __name__,
+    url_prefix="/tags/",
+    permission=perm_models.Permissions.MANAGE_OFFERS_AND_VENUES_TAGS,
+)
+
+
+@tags_blueprint.route("", methods=["GET"])
+def index() -> utils.BackofficeResponse:
```

Comment:
```suggestion
def list_tags() -> utils.BackofficeResponse:
```
Pour être plus proche de `list_offerer_tags` (?)